components/eamxx/src/physics/shoc/impl/shoc_pblintd_impl.hpp

```diff
@@ -78,7 +78,7 @@ void Functions<S,D>::pblintd(
 
   // Initialize
   bool check = true;
-  s_rino(nlev-1) = 0;
```

Comment:
You can remove the `s_rino` variable above.

Reply:
Good catch, thx

Reply:
The `s_rino` var came back in the last commit.

Reply:
It needed to. Now that the full init is only in debug, the original init had to be restored.

Reply:
oh of course.